### DISCUSSION TITLE
Spam bounces

### DISCUSSION THREAD

**Discussion by maticboncina** [simple-login/app#1027] [Self-hosting question] [answered]
Hello,

I have set my IP address to include reverse DNS for my installation of SimpleLogin (app.redacted.com to the IP address), however, when the emails get redirected, I still see "noRDNS" spam flags in the reject log.

  1.8 HTML_FONT_TINY_NORDNS  Font too small to read, no rDNS
  2.0 RDNS_NONE              Delivered to the internal network by a host with no rDNS
  
I assume that I can still fix these two (eg doesn't require the sender to revise the message, more so me confirming DNS correctly).
  
Any help in this area would be much appreciated. 
  
All the best, Matic

**Comment by 0xspade** [comment, marked as answer]
Hi,

Just try "redacted.com" only for your rDNS, but it should also set in the docker though but I dont know how to do that?

it should also change in /etc/hostname and /etc/hosts

**Reply by maticboncina** [reply]
Thank you :)
